assets/payment/settings_script.js

```diff
@@ -0,0 +1,286 @@
+/* global settings_object */
+/* exported redirectToCallback */
+let BuyLicenseWindow = null;
+let UpdatePlanWindow = null;
+let CancelSubscriptionWindow = null;
+let wait = false;
+/* eslint-disable */
+function redirectToCallback(param) {
+	window.location.href += "&" + param;
+}
+function cancelPayment(){
+	let CurrentWindow;
+	let template;
+	if (BuyLicenseWindow && !BuyLicenseWindow.closed) {
+		CurrentWindow = BuyLicenseWindow;
+		template = 'payment_template';
+	} else if (CancelSubscriptionWindow && !CancelSubscriptionWindow.closed) {
+		CurrentWindow = CancelSubscriptionWindow;
+		template = 'cancel_template';
+	}
+	wp.ajax
+	.send(template, {
+		type: 'POST',
+		data: {
+			_ajax_nonce: settings_object.ajaxnonce,
+			priceinfo: JSON.stringify(settings_object.infoplans),
+		},
+	})
+	.done((response) => {
+		CurrentWindow.document.body.innerHTML = '';
+		CurrentWindow.document.write(response);
+		wait = false;
+	});
+}
+function confirmPlanChange(amountPreview, wantedPlanRank){
+	let CurrentWindow;
+	if (BuyLicenseWindow && !BuyLicenseWindow.closed) {
+		CurrentWindow = BuyLicenseWindow;
+	} else if (CancelSubscriptionWindow && !CancelSubscriptionWindow.closed) {
+		CurrentWindow = CancelSubscriptionWindow;
+	}
+	confirmPlanChangeWindow(CurrentWindow, amountPreview, wantedPlanRank);
+}
+//this function is used by the dialog CustomFormWindow 
+function customStripeForm(planRank){
```

Comment:
Codacy found an issue: ['customStripeForm' is defined but never used.](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5907666)